src/User/ProfilePage.jsx

```diff
@@ -29,17 +38,47 @@ const ProfilePage = ({ error, isLoading, user }) => {
     return <p>An error has occurred...</p>;
   }
 
-  const { id, username, firstName, lastName, email, occupation } = user || {};
+  const { id, username, firstName, lastName, email, occupation, managerId } =
+    user || {};
+
+  const displayManager = () => {
+    if (String(managerName) === "None Assigned") {
```

Comment:
what type is managerName, shouldn't it already be a string?

Reply:
This should probably be a constant imported/null etc. so you cant fall out of sync

Reply:
It's a string, I was getting an error at one point that required me to do that `String()` thing but having just removed it I'm not anymore...

Reply:
I'm sad that you guys didn't use Typescript 😔